### PR TITLE
clarify what input the package handles (issue #30)

### PR DESCRIPTION
**Pull request by waldyrious**
/cc @alexanderwallin

### CODE REVIEW

**Comment by alexanderwallin** [comment]
👌

Perhaps a link to https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths or some other source?

File: README.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ svgpath
 [![Coverage Status](https://img.shields.io/coveralls/fontello/svgpath/master.svg?style=flat)](https://coveralls.io/r/fontello/svgpath?branch=master)
 
 
-> Low level toolkit for SVG paths transformations. Sometime you can't use
-`transform` attributes and have to apply changes to svg paths directly.
+Low level toolkit for SVG paths transformations.
```
Comment (puzrin) [inline README.md]: Please keep formatting. Return `> ` to start of line

Comment (waldyrious) [inline README.md]: Oh, yes, I forgot to mention that. With the `>`, that description becomes greyed out and it's a little hard to read (especially for what's supposed to be the main description of the repo). Perhaps some different formatting could be used, like italics?

**Comment by waldyrious** [comment]
@alexanderwallin good point. Linking to https://www.w3.org/TR/SVG11/paths.html#PathData may be more appropriate, agreed?

File: README.md (hunk 1 above)
Comment (puzrin) [inline README.md]: No. Please, keep old style.

**Comment by puzrin** [comment]
No need to make things complex. Just modify example to use real string and show result.

**Comment by alexanderwallin** [comment]
I would appreciate that link.

File: README.md (hunk 1 above)
Comment (waldyrious) [inline README.md]: Ok, sure.

**Comment by waldyrious** [comment]
I've restored the blockquote formatting, as requested. I also added the link to the docs -- I agree with @alexanderwallin that it is useful to end-users.

**Comment by puzrin** [comment]
Ok. Please, join all to single comit and i will merge it

**Comment by waldyrious** [comment]
That can be done directly on github now :) just use the "Squash and merge" option, like this:

screenshot from 2017-06-25 16-31-33

**Comment by puzrin** [comment]
That leave garbage in comment lines. Could you join manually and force push? Text - something like "Clarify description in readme"

**Comment by waldyrious** [comment]
Oh, so you tried it before? You might want to check it out again -- the functionality probably has been changed since you last used it, because now it does allow editing the full commit message (including the extended description) before merging, directly from the web interface. It's quite convenient, really :) But I can do it myself in the command line if you prefer.

**Comment by puzrin** [comment]
Seems to work. Thank you for info, now web squash is really useable.

**Comment by waldyrious** [comment]
Yeah, they've been making several nice improvements to the UI recently, which makes both contribution and maintenance easier :)

**Pull request merged**